lms/djangoapps/instructor_task/api.py

```diff
@@ -337,7 +337,7 @@ def submit_calculate_students_features_csv(request, course_key, features):
     """
     task_type = 'profile_info_csv'
     task_class = calculate_students_features_csv
-    task_input = {'features': features}
+    task_input = features
```

Comment:
@douglashall , is this sufficient to get the task input under 255 characters? Why didn't you stick with your old fix of moving the feature selections to inside the task?